src/data-table/ContextualMenu.tsx

```diff
@@ -64,10 +64,6 @@ export function ContextualMenu<T extends ReferenceObject>(props: ContextualMenuP
                 left: positionLeft,
                 top: positionTop,
             }}
-            anchorOrigin={{
-                vertical: "center",
-                horizontal: "center",
-            }}
```

Comment:
Removed this because it is ignored when working with ``anchorPosition`` (see https://material-ui.com/api/popover)